### DISCUSSION TITLE
Any plans to include TV/Movies translated title in services/scrapping ?

### DISCUSSION THREAD

**Discussion by philamp** [itsToggle/plex_debrid#247] [Ideas]
Sometimes "local" trackers are naming foreign movies with a totally different string or just the locally translated string 

(for instance in french TMDB: 
- "the hangover" becomes "very bad trip" 
- "the fifth element" becomes "le cinquieme element" 
) 

so if scrapping searches only in english, some trackers won't return any results when movie title is different in the local language.

Do you think you will include translated titles in TV/Movie content/services (jellyseer etc...) and as well in scrapping (prowlar, jacket etc..) ?

Thanks

**Comment by itsToggle** [comment]
Hi! This is already implemented :) I havent documented this feature yet, as its not yet really polished, but you can already try it out:

If you connect plex_debrid to trakt.tv (/Settings/Content Service/Trakt/users/add user), plex_debrid will automatically scrape for alternative movie titles like the ones youve mentioned. plex_debrid relies on the trakt "alias" database for movie/show titles for this. These aliases only include alternative english titles by default. 

To look for non-english movie titles first, you can define the prefered scraping language inside the "version" settings. To do this, go to /Settings/Scraper Settings/versions/ and edit one of the versions that plex_debrid should look for (or create a new one). While editing a version, you can define its scraping language by typing "lang". You will then be prompted to enter a 2-letter language code ("en"=englisch, "fr"=french, etc.)

I will document this feature on the github readme aswell! :)

**Reply by philamp** [reply]
Well, great, but I have one question left : If I take an example:

1/ I look for "le cinquieme element" (french title) on trakt, he knows this movie **but unfortunately only with it english name** (whereas with TMDB it works with any language)
2/ Ok I know its english name so I eventually find it in trakt
3/ I click "add to watchlist" in trakt
4/ Plex_Debrid detects this movie has just been added
**5/ does it fetch it in the TMDB to get its french title ?**
6/ It scraps it with rarbg, jackett or prowlarr ....

**Reply by itsToggle** [reply]
plex_debrid is using the trakt api to get known alternative titles for movies, not TMDB :)

**Reply by benten3** [reply]
Hi thanks for this explanation.

Can you confirm that at this moment we can't do the same thing with "plex watchlist" because the metadata api only return english movie titles ?

The only way to scrap with alternatives movies titles is to use trakt ?

Thanks !

**Comment by Charlotte-br560** [comment]
Absolutely! Incorporating translated titles for better search results, especially considering variations across different languages, sounds like a great enhancement. It would definitely improve the user experience and accessibility.
